package.json

```diff
@@ -172,7 +172,7 @@
     "dockerode": "^2.5.8",
     "dockerode-options": "^0.2.1",
     "ejs": "^2.5.7",
-    "etcher-sdk": "^2.0.5",
+    "etcher-sdk": "^2.0.14",
```

Comment:
This change is in a separate commit. It's not completely random that this commit is bundled in this PR, because the the `balena-cli` issue (#1419) resolved by this PR regarded the `balena local flash` command, with an error that pointed to the `xxhash` native module, which is a dependency of [etcher-sdk](https://github.com/balena-io-modules/etcher-sdk). As it happens, `etcher-sdk` v2.0.14 bumped `xxhash` in relation to Node 12 support.

Note that, as indicated in the `npm-shrinkwrap.json` file, the actual `etcher-sdk` version bump was from `2.0.13` to `2.0.14`, rather than from `2.0.5` to `2.0.14` as `package.json` might lead you to believe. The `etcher-sdk` [changelog](https://github.com/balena-io-modules/etcher-sdk/blob/master/CHANGELOG.md) for `2.0.14` points only to the `xxhash` bump, hence this commit / PR also being of type `patch` rather than `minor`.